backend/yonder/views.py

```diff
@@ -169,3 +173,33 @@ def put(self, request, *args, **kwargs):
     @swagger_auto_schema(tags=['comments'])
     def delete(self, request, *args, **kwargs):
         return self.destroy(request, *args, **kwargs)
+
+class inbox(generics.ListCreateAPIView):
+
+    @swagger_auto_schema(tags=['inbox'])
+    def get(self, request, *args, **kwargs):
+        # if not request.user.is_authenticated:
+            # return Response(status=status.HTTP_401_UNAUTHORIZED)
+
+        inbox = get_object_or_404(Inbox.objects.all(), author_id=kwargs["author_id"])
+        author = get_object_or_404(Author.objects.all(), id=kwargs["author_id"])
+        data = {
+            "type": "inbox",
+            "author": author.get_absolute_url(),
+            "items": inbox.items
+        }
+        return Response(data, status=status.HTTP_200_OK)
```

Comment:
This works, but we are required to paginate it. Look into using `pagination_class`, it's mentioned [here](https://www.django-rest-framework.org/api-guide/generic-views/#genericapiview).

Reply:
So I didn't user `pagination_class` because it would paginate the actual inbox, but we want to paginate the items, so im using this pagintor https://docs.djangoproject.com/en/3.1/topics/pagination/

Reply:
Ah, ok that makes sense. Looks good, good work.